src/app/banners/BannerWrapper.tsx

```diff
@@ -26,7 +26,7 @@ const BannerWrapper = (): JSX.Element => {
 
   useEffect(() => {
     bannerManager.handleBannerDisplay('free', setShowBanner);
-    bannerManager.handleBannerDisplay('subscription', setShowSubscriptionBanner);
+    bannerManager.handleBannerDisplay(plan.individualSubscription?.type ?? 'free', setShowSubscriptionBanner);
```

Comment:
The logic of the BannerWrapper and the BannerManager is very confusing for me, are sending here the type of the subscription and checking in the BannerManager the same. 
For future updates of this we will require tests in order to be able to add changes related to banner managment.